src/commands/remember/remember-chunk.js

```diff
@@ -20,25 +62,38 @@ module.exports = {
         },
         {
             name: 'speed',
-            description: 'Speed of the Search. Lower value is more accurate but slower.',
+            description: 'Speed of the Search. Lower value is more accurate but slower. Default is 50',
             type: ApplicationCommandOptionType.Number,
         },
+        {
+            name: 'exclude-bot-messages',
+            description: `If bot messages should be excluded in the message collection. Default is ${defaultExcludeBotMessages}`,
+            type: ApplicationCommandOptionType.Boolean
+        }
     ],
 
     //logic, 
     callback: async(client, interaction) =>{
         try{
             await interaction.deferReply();
 
-            //get the id    
-            const value = interaction.options.get('minutes').value;
+            //get the hours   
+            const valueHours = interaction.options.get('hours').value;
+
+            //get the minutes   
+            const valueMinutes = interaction.options.get('minutes').value;
+
+            //bot messages
+            const excludeBotMessages = interaction.options.getBoolean('exclude-bot-messages') ?? defaultExcludeBotMessages;
 
             //validate accuracy
-            let accuracy = interaction.options.getNumber("speed") ?? 100;
+            let accuracy = interaction.options.getNumber("speed") ?? 50;
             if(accuracy < 25) accuracy = 25; 
             if(accuracy > 100) accuracy = 100;
 
-            const numberOfMinutes = (value > 600 ? 600 : value).toString() + 'm'; //formatting for ms
+            //format and clamp
+            const numberOfHours = (valueHours > 5 ? 5 : valueHours).toString() + 'h'; //formatting for ms
+            const numberOfMinutes = (valueMinutes > 59 ? 59 : valueMinutes).toString() + 'm'; //formatting for ms
```

Comment:
For the clamping above you could also do something like `Math.min(valueMinutes, 59)` or `accuracy = Math.min(accuracy, 100)`.